### DISCUSSION TITLE
💋 【共学营打卡】【第2次 Milestone】【Move-0x02】Compile and deploy My-Counter contract

### DISCUSSION THREAD

**Discussion by 99Kies** [NonceGeek/Move-Camp#109] [Projects-Others]
教程见：
https://github.com/NonceGeek/Web3-dApp-Camp/tree/main/move-dapp#13-contract-framework

回复截图打卡

**Comment by qiwihui** [comment]
<img width="1150" alt="Screen Shot 2022-07-27 at 4 56 19 PM" src="https://user-images.githubusercontent.com/3297411/181206399-5571f202-1342-4284-a946-c2ad18dd8e86.png">
<img width="1553" alt="Screen Shot 2022-07-27 at 4 55 34 PM" src="https://user-images.githubusercontent.com/3297411/181206386-96070c70-040d-472e-8d3e-e03076984449.png">

**Reply by leeduckgo** [reply]
First Kill: ¥88.88

**Comment by leeduckgo** [comment]
![](https://camo.githubusercontent.com/2a46dae68ba79fd9eeb80fa24327efdcfc8e4ce981cf2ec0bd4644a57392df71/68747470733a2f2f747661312e73696e61696d672e636e2f6c617267652f653663396432346567793168346c647a386a64376c6a32313373306c6d6a75352e6a7067)

**Comment by 99Kies** [comment]
![image](https://user-images.githubusercontent.com/45918704/181441863-dc874d55-2456-41d4-a001-7d889329b458.png)
![image](https://user-images.githubusercontent.com/45918704/181442717-a7beae0c-5acd-46ea-a865-6ae781f84acb.png)

**Comment by hqwangningbo** [comment]
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/57781136/181446019-7d19b2a7-337d-4248-b611-96843118261e.png">

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/57781136/181446283-1c0a5391-7fe6-4e67-93da-a794636b9579.png">

**Comment by pochenai** [comment]
<img width="618" alt="截屏2022-07-29 下午7 52 27" src="https://user-images.githubusercontent.com/13568446/181754604-b2a951e1-3dbb-45c7-9f48-1a4550a4a3d7.png">
![Uploading 截屏2022-07-29 下午8.00.11.png…]()
<img width="617" alt="截屏2022-07-29 下午8 03 13" src="https://user-images.githubusercontent.com/13568446/181754628-7b3615ab-43c4-44d3-a913-6eaa6c0ad4e2.png">

**Comment by 0xKT** [comment]
<img width="1222" alt="image" src="https://user-images.githubusercontent.com/74288668/182537173-ec2b52d3-5f34-4fe8-a71a-b99840d575ca.png">
<img width="1233" alt="image" src="https://user-images.githubusercontent.com/74288668/182537242-86eb94bf-da60-4769-9e5d-950e3f005c3c.png">

**Comment by TecSong** [comment]
<img width="1141" alt="image" src="https://user-images.githubusercontent.com/8627464/195105158-3ee261c2-c281-4a45-a951-2deb824e89e3.png">

**Comment by nishuzumi** [comment]
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/13927203/195113226-c3975c55-7f0b-4259-bfe8-147d2f268323.png">
<img width="1199" alt="image" src="https://user-images.githubusercontent.com/13927203/195113338-69b6d1d9-a879-416e-8651-0f1d1ce637a5.png">

**Reply by cradle0fFilth** [reply]
![move1](https://user-images.githubusercontent.com/64477775/195132374-575be5ae-143f-451c-94eb-52ee401d5c72.PNG)
![move2](https://user-images.githubusercontent.com/64477775/195132396-db908b61-369f-4107-88ea-a5050b0eb2af.PNG)

**Comment by shing19** [comment]
<img width="569" alt="Screen Shot 2022-10-12 at 00 24 34" src="https://user-images.githubusercontent.com/38907754/195147490-5080a9ab-27a4-4c47-a3c8-59f69943a012.png">
<img width="1223" alt="Screen Shot 2022-10-12 at 00 25 32" src="https://user-images.githubusercontent.com/38907754/195147501-545bf9d5-6074-4aff-aa53-f063001ce1b4.png">

**Comment by sharkqwy** [comment]
<img width="729" alt="截屏2022-10-17 15 21 22" src="https://user-images.githubusercontent.com/1372795/196113591-dceabd9c-183e-488c-8789-dafb845f74a0.png">

**Comment by zhaoge5859** [comment]
打卡

<img width="909" alt="截屏2022-11-01 20 18 09" src="https://user-images.githubusercontent.com/116735858/199243516-f293cdf5-2998-4d0c-9017-4ef623209737.png">
<img width="993" alt="截屏2022-11-01 21 21 33" src="https://user-images.githubusercontent.com/116735858/199243541-be4b7070-3cb2-4b60-96a7-c0c304fc21f7.png">

**Comment by pingp76** [comment]
![WX20221106-205743@2x](https://user-images.githubusercontent.com/8874696/200172230-e295994d-1ab0-4190-bf21-dbeef79f15d2.png)
mycounter那个很早就试过了，我贴一个显示我在本地自己部署的发币合约

**Comment by MagicGordon** [comment]
<img width="1675" alt="7062" src="https://user-images.githubusercontent.com/19465870/200176288-e9c36aa6-e5d5-4b42-9e08-767be5b31bff.png">

**Comment by Hosuke** [comment]
> https://github.com/WeLightProject/Web3-dApp-Camp/blob/main/move-dapp/CryptoXiuXian%20in%20Move%20Lang.md#13-contract-framework

更新有效连接🔗
https://github.com/NonceGeek/Web3-dApp-Camp/tree/main/move-dapp#13-contract-framework

**Comment by Hosuke** [comment]
<img width="1365" alt="image" src="https://user-images.githubusercontent.com/6568873/200272930-68e034fa-6f15-487d-a11d-9b4c49be7cbd.png">

**Comment by v1xingyue** [comment]
![image](https://user-images.githubusercontent.com/974169/200278851-4d7b4295-6d1a-4b0c-ad48-f706d4ad9f62.png)

编译小工具: 

```shell
mpm release 
blob_path=$(ls release/*.blob)
console.sh dev deploy $blob_path -s 0xC13Ba46E31705e1A5B30Bc0BfA564824 -b
```

**Comment by xiegudong45** [comment]
![image](https://user-images.githubusercontent.com/30063330/200348314-9612082a-acb5-4037-8f53-5b6f16d26c53.png)
![image](https://user-images.githubusercontent.com/30063330/200348051-02374883-8667-4a13-aa3d-c3550f5105a3.png)

**Comment by Demian101** [comment]
<img width="828" alt="image" src="https://user-images.githubusercontent.com/33189338/200459894-ef330770-706d-49bb-a5ce-601cdbe93ffe.png">

**Comment by livelybug** [comment]
![image](https://user-images.githubusercontent.com/13627060/200784818-b28841e5-953c-45b0-847c-a86d9376a0dd.png)

**Comment by valerywai** [comment]
![image](https://user-images.githubusercontent.com/53373706/200794892-c66a8352-9a27-4ae7-b550-1c3b364689c2.png)
![image](https://user-images.githubusercontent.com/53373706/200795539-fccc479f-f40c-4f21-95d7-68f663ccf107.png)

**Comment by geometryolife** [comment]
**编译**：
![0](https://user-images.githubusercontent.com/54882546/201573085-37c84c15-1975-4fbb-bb6c-a26a26dd40ea.png)
**部署**：
![1](https://user-images.githubusercontent.com/54882546/201572854-6ee3186b-597b-4508-acbd-b173d2dd5ab5.png)
**调用**：
![2](https://user-images.githubusercontent.com/54882546/201573153-7876c01d-f4cf-4172-a94e-5ac85e75a300.png)

**Comment by hi-chenerge** [comment]
![release](https://user-images.githubusercontent.com/82131587/211197682-aeed363c-6a04-4423-81db-e75c57c5bb65.jpg)
![deploy](https://user-images.githubusercontent.com/82131587/211197686-887637c4-54d8-4bc3-a444-de6eaa13e267.jpg)
![init](https://user-images.githubusercontent.com/82131587/211197688-b89bd6f7-6737-4958-994c-2e99f63cd26b.jpg)
![getAndIncr](https://user-images.githubusercontent.com/82131587/211197692-23e15c8b-57be-45bc-91dc-5e79184b0e7a.jpg)
![get](https://user-images.githubusercontent.com/82131587/211197694-f6c377e0-6761-4b9e-a594-2bca8f179c38.jpg)

**Reply by hi-chenerge** [reply]
release - deploy - init - get and incr - get

**Comment by 0xsu** [comment]
<img width="632" alt="image" src="https://user-images.githubusercontent.com/101964290/212122985-252f7518-a278-4267-9042-c76dd74c627a.png">

**Comment by Whitehare2023** [comment]
![9f1a70ea52bf13f336ce09159fa748e](https://user-images.githubusercontent.com/124434634/216923294-2a703370-1ea0-461f-92a5-107aaa10b250.png)
![bcb6cb5b8446aed45737b96e4c623e3](https://user-images.githubusercontent.com/124434634/216923313-ed0e2343-21b2-4e6c-95b7-23b67534163f.png)

**Comment by baidang201** [comment]
![image](https://user-images.githubusercontent.com/3405789/218302497-1237290c-bc43-4e2d-87c0-7e6ed1a39507.png)
![image](https://user-images.githubusercontent.com/3405789/218302519-714d6d4c-dfd6-43d4-90ea-ca4a0496d9db.png)

**Comment by DivenYoung** [comment]
<img width="1131" alt="CleanShot 2023-02-12 at 19 17 11@2x" src="https://user-images.githubusercontent.com/22409395/218308675-1921fc48-7675-46ee-9a1e-92b5430dabca.png">
